test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -0,0 +1,392 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Frameworks;
+using NuGet.Packaging;
+using NuGet.Test.Utility;
+using NuGet.Versioning;
+using NuGet.XPlat.FuncTest;
+using Xunit;
+
+
+namespace NuGet.Commands.FuncTest
+{
+    [Collection("Dotnet Integration Tests")]
```

Comment:
This is how CollectionFixtures work :
Collection Fixtures
When to use: when you want to create a single test context and share it among tests in several test classes, and have it cleaned up after all the tests in the test classes have finished.

Sometimes you will want to share a fixture object among multiple test classes. The database example used for class fixtures is a great example: you may want to initialize a database with a set of test data, and then leave that test data in place for use by multiple test classes. You can use the collection fixture feature of xUnit.net to share a single object instance among tests in several test class.

To use collection fixtures, you need to take the following steps:

Create the fixture class, and put the the startup code in the fixture class constructor.
If the fixture class needs to perform cleanup, implement IDisposable on the fixture class, and put the cleanup code in the Dispose() method.
Create the collection definition class, decorating it with the [CollectionDefinition] attribute, giving it a unique name that will identify the test collection.
Add ICollectionFixture<> to the collection definition class.
Add the [Collection] attribute to all the test classes that will be part of the collection, using the unique name you provided to the test collection definition class's [CollectionDefinition] attribute.
If the test classes need access to the fixture instance, add it as a constructor argument, and it will be provided automatically.

Source : http://xunit.github.io/docs/shared-context.html